rest-api-spec/src/yamlRestCompatTest/resources/rest-api-spec/api/cluster.post_voting_config_exclusions_with_node_name_part.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Since this route will always produce an error even in compatibility mode, should we not create an API specification for it? I'm not sure we want this API to be in generated clients

Reply:
Since it's under `rest-api-spec/src/yamlRestCompatTest/resources/rest-api-spec/api/`, I think we're already good on that, right? That is, this is just a spec definition for the purposes of the yml tests themselves, it's not supposed to leak up and out.

Reply:
Hmm, I don't know the rules for which files are picked up by the artifacts API and which aren't. If this file doesn't make it's way into the built artifact for `rest-api-spec` then this is fine.

Reply:
I am pretty confident that this will not get picked up to be visible by clients, if it does we can quickly fix it (or revert this PR).